lib/saml/user_attributes/ssoe.rb

```diff
@@ -96,7 +99,16 @@ def mhv_correlation_id
       end
 
       def mhv_account_type
-        safe_attr('va_eauth_mhvassurance')
+        val = safe_attr('va_eauth_mhvassurance')
+        val ||= safe_attr('va_eauth_credentialassurancelevel') if csid == 'mhv'
+        # FIXME: for inbound SSOe what number indicates Advanced/Basic?
```

Comment:
Both Basic and Advanced would map to `credentialassurancelevel=1`

Reply:
So we have a loss of fidelity here...BUT, there's a separate thread with the MHV team going where we would do away with a lot of  the account-level-conditional logic so 🤞 that this wouldn't be an issue.

Reply:
I'm wondering if we can just return nil here for inbound. The mhv_account_type is used in a few places:
- to populate the `sign_in:account_type` object which is used for the MHV shared cookie which we don't use in SSOe.
- to populate `user.mhv_account_type` which is used for various service authorization checks, but that has a fallback to getting the account type from the MHV API.
- wherever else sign_in might be serialized into the user profile for front end, I haven't checked this part.

Reply:
If we return `nil` here for MHV inbound users, `loa_highest` will always evaluate to 1.  Is that value only used for upleveling or does the vets-website code use that value for something?

Reply:
@patrickvinograd looks like vets-website is using the account type here https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/site-wide/cta-widget/helpers.js#L37-L52